components/modal/core/Dialog.razor.cs

```diff
@@ -138,20 +138,20 @@ private void OnDialogMouseDown()
             _dialogMouseDown = true;
         }
 
-        private async Task OnMaskMouseUp()
```

Comment:
I'm not sure why Antd listens to the mouseup event to determine if dialog was clicked.

It seems that it's OK to reset _dialogMouseDown in MaskClick, no matter whether the sub DOM is clicked or not, the mask click event is always triggered, and it is behind mouseup event. So I remove the MouseUp event of dialog-wrap DOM.